### PR TITLE
3.0.0 Update

### PR DESCRIPTION
**Pull request by GanstaKingofSA**
This PR adds the following functionality

## Features
- Added Uncensored Mode into the mod itself to better handle sensitive/adult content in mods
- Added Let's Play Mode to censor names and other content that may harm streamers/doki-tubers content. 
    > A automatic detection has been added on startup to enable this mode if streaming/recording software is running (based off DDLC's stream list). This will only happen in the Extra Settings from this update are enabled to be viewed from the settings page.
- Gallery Menu
- ICO Creation Support for Windows
- Better comments on DDLC's code lines
- Username and Process List support for Linux/Mac OS.
- Function to check for processes running on the system for scenes like Act 3, etc.
    > I still highly recommend using Let's Play Mode instead if you are checking if the person is streaming/recording.

## Fixes
- Removed `scripts.rpa` requirement.
- Fixed Ren'Py 6 poem game error.
- Fixed the process list and BSOD screen for `wmic`'s depreciation in Windows 11 > build 22000.
- Fixed a bug where a `future.standard_library` import error will happen on Linux due to old Ren'Py 6 libraries.

### CODE REVIEW

**Comment by GanstaKingofSA** [comment]
PR #14 is considered to be a part of the 2.5.0 update but separate branch as this is experimental ATM.

**Comment by GanstaKingofSA** [comment]
This PR now adds these fixes

- Fixed the BSOD screen and splash screen process list commands due to `wmic` depreciation for Windows 10 and Windows 11 builds after (but possibly including) Version 21H1.
- Rewrote some DDLC commands to be more integrated and depreciated some in return.

File: game/splash.rpy [hunk 1]
```diff
@@ -375,7 +375,7 @@ label splashscreen:
         # software so the game can enable Let's Play Mode automatically and notify
         # the user about it if extra settings are enabled.
         if extra_settings:
-            if process_check(["obs32.exe", "obs64.exe", "obs.exe", "xsplit.core.exe", "livehime.exe", "pandatool.exe", "yymixer.exe", "douyutool.exe", "huomaotool.exe", "BlueTool"]):
```
Comment (Dobby233Liu) [inline game/splash.rpy]: What was this "BlueTool"

Comment (GanstaKingofSA) [reply game/splash.rpy]: Leftover Test Code for running the process list on MacOS.

**Pull request merged**